holoviews/plotting/bokeh/element.py

```diff
@@ -512,7 +552,7 @@ def initialize_plot(self, ranges=None, plot=None, plots=None, source=None):
         self.handles['plot'] = plot
 
         # Get data and initialize data source
-        empty = self.callbacks and self.callbacks.downsample
+        empty = False
         if self.batched:
```

Comment:
Is this temporary? If not, there is no reason for ``empty`` to appear as I don't see it being set a different value anywhere...